src/game/common/ini/ini.h

```diff
@@ -150,6 +152,7 @@ class INI
     static void Parse_Acceleration_Real(INI *ini, void *formal, void *store, const void *user_data);
     static void Parse_Bit_In_Int32(INI *ini, void *formal, void *store, const void *user_data);
     static void Parse_And_Translate_Label(INI *ini, void *formal, void *store, const void *user_data);
+    // Passed string array must be null terminated!
```

Comment:
If you are adding these comments why not add one to the field parse functions as well L99